.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.19-openshift-4.13
+  tag: rhel-8-release-golang-1.17-openshift-4.11
```

Comment:
it looks like we are undoing a recent ART commit with this

Reply:
Good catch! Weird it kept only the latest [UPSTREAM: <carry>: Updating ose-cluster-kube-cluster-api-operator ima…](https://github.com/openshift/cluster-api-operator/commit/c23bc67c8dd546ad6ed28d19a07b33fc9e79d54a) commit, but not the one prior, which updated this file to 1.19. Will bump it manually.

Reply:
Bumped